src/components/ActionControls/ActionControls.js

```diff
@@ -54,11 +54,42 @@ class ActionControls extends React.Component<Props, State> {
         const TypeControls = () => {
             switch (type) {
                 case TYPES.highlight:
+                    if (!canComment && !canDelete) {
+                        return null;
+                    }
+
+                    return (
+                        <HighlightControls
+                            canAnnotateAndDelete={canDelete}
+                            canComment={canComment}
```

Comment:
seems like `canComment` will always be false here, could we omit it to have the same effect?